NumberBaseball/Source.c

```diff
@@ -0,0 +1 @@
+#include <stdio.h>
```

Comment:
Delete this file

Reply:
Why get rid of stdio.h? if i don't have that then i can't use sscanf_s right?